casper/src/main/scala/coop/rchain/casper/util/comm/ApproveBlockProtocol.scala

```diff
@@ -70,7 +70,7 @@ object ApproveBlockProtocol {
       duration: FiniteDuration,
       interval: FiniteDuration): F[ApproveBlockProtocol[F]] =
     for {
-      now   <- Timer[F].clockRealTime(MILLISECONDS)
+      now   <- implicitly[Timer[F]].clock.realTime(MILLISECONDS)
```

Comment:
no :) I prefer the previous version any time :)

Reply:
I do too :) Timer is though no longer considered a typeclass by the cats people, and as part of that it no longer has an `apply` method.

Reply:
I'm pretty sure it is 'don't use implicits to pass instances of non-typeclasses'. Also, I'm really tired after fixing all those conflicts and would prefer nits like 'pass it via methods args / implicit arg list' to be left for future PRs :)

Reply:
Ok, it needs to stay as is.